relayer/chains/wasm/types/types.go

```diff
@@ -71,11 +73,11 @@ type EventsList struct {
 type TxResultResponse struct {
 	Height int64 `json:"height"`
 	Result struct {
-		Code      int     `json:"code"`
-		Codespace string  `json:"codespace"`
-		Data      []byte  `json:"data"`
-		Log       string  `json:"log"`
-		Events    []Event `json:"events"`
+		Code      int              `json:"code"`
+		Codespace string           `json:"codespace"`
+		Data      []byte           `json:"data"`
+		Log       string           `json:"log"`
+		Events    []abiTypes.Event `json:"events"`
```

Comment:
Please cleanup the previously added struct types since we are shifting to one from the lib.